python/openmldb/sdk/sdk.py

```diff
@@ -127,6 +127,11 @@ def execute_sql(self, db, sql):
             return False, status.msg
         else:
             return True, rs
+        
+    def is_Online_Mode(self):
```

Comment:
Rename the function name such as `isOnlineMode()`.

Reply:
updated.